demos/collection/grid.php

```diff
@@ -72,6 +74,45 @@
 // $grid->addExecutorButton($deleteExecutor, new Button(['icon' => 'times circle outline']));
 
 $sel = $grid->addSelection();
+// Executing a modal on a bulk selection
+$callback = function ($m, $ids) use ($grid) {
```

Comment:
types should be defined, `$m` variable should be named `$modal`

Reply:
done